plaidml/edsl/edsl.h

```diff
@@ -1161,6 +1169,7 @@ inline Tensor Tensor::operator!() const { return intrinsicCall("logical_not", {*
   inline Tensor operator _op_(uint64_t lhs, const Tensor& rhs) { return intrinsic(_fn_, lhs, rhs); }      \
   inline Tensor operator _op_(double lhs, const Tensor& rhs) { return intrinsic(_fn_, lhs, rhs); }
 
+// clang-format off
```

Comment:
What's this for?

Reply:
For some reason clang-format wanted to add a space here (between `OPS` and `(`:
`PLAIDML_EDSL_DEFINE_TENSOR_BINARY_OPS (^, "bit_xor");`
which was then caught by cpplint, preventing me from being able to push my commit. If there's a preferred way to handle this lmk

Reply:
Make sure you use the `.cenv` conda environment, it has a specific version of clang-format that we should all be using.